grove.tm1637.device.lib.nut

```diff
@@ -0,0 +1,190 @@
+// MIT License
+
+// Copyright 2017 Electric Imp
+
+// SPDX-License-Identifier: MIT
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO
+// EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+// OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+// ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+// OTHER DEALINGS IN THE SOFTWARE.
+
+const GROVE_TM1637_HIGH = 1;
+const GROVE_TM1637_LOW = 0;
+const GROVE_TM1637_ADDR_AUTO = 0x40;
+const GROVE_TM1637_ADDR_FIXED = 0x44;
+const GROVE_TM1637_DATA_CMD = 0xC0;
+const GROVE_TM1637_DISPLAY_CMD = 0x88;
+
+class GroveTM1637 {
+
+    _dataPin = null;
```

Comment:
Does it need to have a `static VERSION`?

Reply:
Good point - totally forgot about that. Will add it in.